Training/Program.cs

```diff
@@ -1,7 +1,79 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
+﻿//Numbers to words or roman numbers
```

Comment:
```suggestion
﻿// Numbers to words or roman numbers
```
Leave a space after //